Fulfillment/APIs/Outstanding orders.md

```diff
@@ -0,0 +1,169 @@
+# Outstanding Orders API
+
+Fetches a comprehensive list of all orders that are brokered at a facility for fulfillment. To get the Outstanding orders, you will need to call the /solr-query endpoint with the POST method. 
+
+## Request
+
+### Endpoint
+
+`https://<host>/api/solr-query`
+
+Example: https://demo-oms.hotwax.io/api/solr-query
+
+### Header
+
+#### Body
+
+```
+{
+  "json": {
+    "params": {
+      "rows": 1,
+      "group": true,
+      "group.field": "orderId",
+      "group.limit": 1000,
+      "group.ngroups": true,
+      "defType": "edismax",
+      "q.op": "AND",
+      "qf": "orderId"
+    },
+    "query": "(* *)",
+    "filter" : [
```

Comment:
The way filters are written here is different from the inprogress page. I like this format better because its easier to read, we should update the other ones to match this as well

Reply:
updated the other ones